packages/appstore/src/components/add-more-wallets/wallet-add-card.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { TWalletInfo } from 'Types';
+import { Text, WalletCard } from '@deriv/components';
+import { useCurrencyConfig } from '@deriv/hooks';
+import { observer, useStore } from '@deriv/stores';
+import { getWalletCurrencyIcon } from '@deriv/utils';
+import wallet_description_mapper from 'Constants/wallet_description_mapper';
+
+type TAddWalletCard = {
+    wallet_info: React.PropsWithChildren<TWalletInfo>;
+};
+
+const AddWalletCard = observer(({ wallet_info }: TAddWalletCard) => {
+    const {
+        ui: { is_dark_mode_on },
+    } = useStore();
+
```

Comment:
```suggestion
    const { ui } = useStore();
    const { is_dark_mode_on } = ui;
```
Isn’t it more readable? and feets into two lines :)

Reply:
I already made opposite changes :) let's choose one right code style and after this will use it. It's counterproductive to make the opposite changes based on code review suggestions

Reply:
@sergei-deriv @yauheni-deriv I personally like a one-liner :) lol:
`const { ui: { is_dark_mode_on } } = useStore();`